build.xml

```diff
@@ -71,7 +76,7 @@
           resultproperty="test.result"
           failonerror="true">
       <arg value="-c"/>
-      <arg value=". bin/activate;cd ../extension;cfx test ${cfx.binary}"/>
+      <arg value=". bin/activate;cd ../extension;cfx --verbose test ${cfx.binary}"/>
```

Comment:
I would say lets leave verbose logging as hard-coded for the `test` target. If we feel it's wrong we can change it later. So I assume that we can merge this PR as is? If that is the case will you do it? For me it looks fine.